scripts/jenkins/ardana/ansible/run-crowbar-tests.yml

```diff
@@ -25,6 +25,16 @@
           debug:
             msg: "http://{{ ansible_host }}:9091/"
 
+        - name: Ensure temporal repo for python-virtualenv is in place
+          zypper_repository:
+            repo: "http://download.opensuse.org/repositories/devel:/languages:/python:/backports/SLE_{{ \
```

Comment:
For our purposes, you can simplify this to be just:
```
SLE_{{ ansible_distribution_major_version }}_SP{{ ansible_distribution_release }}
```

Reply:
I would prefer to leave it, to be honest, that way it is prepared for SLE_15 and the like.

Reply:
Ok, fair enough.